### DISCUSSION TITLE
rewriting syslog hostname

### DISCUSSION THREAD

**Discussion by peterand-pa** [dmachard/DNS-collector#583] [Q&A]
Has anybody managed to re-write the source hostname when logging dnstap messages to syslog ? 
If so , how  ?
Currently, it gets sent as the hostname of the system running go-dnscollector , which isn't ideal.
I've tried fudging it by sending to tcpclient: instead of syslog: ,  but the text-format string is a bit limited.
Notes:   
1. The "hostname" config variable doesn't seem to do anything at all.
2. Running go-dnscollector 0.40.2

**Comment by denis-machard** [comment]
It's clearly a bug, can you open an issue with your config?

**Reply by peterand-pa** [reply]
Sorry,  I was mistaken when testing.  The "hostname" variable works ,  but it _seems_ you can only have static text.
Does the code handle ( for example )  ? :
hostname: response-ip
, or some other way of putting in where the dnstap message came from ?

**Reply by peterand-pa** [reply]
Note:  If text-format catered for text other than pre-defined variables , I could create my own outbound syslog message including the <PRI>  prefix.
What would be nice if it could do handle something like :
text-format : "blah blah blah {variable1} {variable2} more text {variable3}"

**Reply by dmachard** [reply]
> hostname: response-ip
Not possible right now.

Can you open issues as Feature Request ?  It's more easy to track it.
